packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -132,6 +132,23 @@ export function applySchemaOp(state: FuzzTestState, operation: SchemaChange) {
  */
 export function applyFieldEdit(tree: FuzzView, fieldEdit: FieldEdit): void {
 	const field = navigateToField(tree, fieldEdit.field);
+	if (fieldEdit.constraint !== undefined) {
+		switch (fieldEdit.constraint.content.type) {
+			case "nodeConstraint": {
+				const constraintNodePath = fieldEdit.constraint.content.content;
+				const constraintNode =
+					constraintNodePath !== undefined
+						? navigateToNode(tree, constraintNodePath)
+						: undefined;
+				if (constraintNode !== undefined) {
+					tree.checkout.editor.addNodeExistsConstraint(constraintNode.anchorNode);
```

Comment:
This constraint doesn't do much if we're not in a transaction.

Reply:
Refactored constraint edits to its own generator, and added acceptance condition to check if transaction is in progress